src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
-    }
+    public Color getRandomColor() {
```

Comment:
<img width="855" alt="Screenshot 2022-01-19 at 13 08 27" src="https://user-images.githubusercontent.com/33026471/150185118-8207b96f-3ba4-436f-8823-71970fd0930d.png">
from task description this method should return String, let's fix it. We can also use String type inside Ball class to save color